test/AutoRest.Shared.Tests/RequestContentHelperTests.cs

```diff
@@ -1,149 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text.Json;
-using System.Xml;
-using System.Xml.Linq;
-using NUnit.Framework;
-
-namespace Azure.Core.Tests
```

Comment:
Lets not remove these tests, lets just pull in one instance of the internal class to use as the test source.

This should be happening for you with this https://github.com/Azure/autorest.csharp/blob/feature/v3/test/AutoRest.Shared.Tests/AutoRest.Shared.Tests.csproj#L23.

We definitely want to unit tests the generated classes.

Reply:
introduced back this test case